submap_service: "/elevation_mapping/get_submap"
update_rate: 2.0
robot_frame_id: "base"
map_frame_id: "map"
map_center_x: 0
map_center_y: 0
map_length_x: 6.0
map_length_y: 6.0
threshold: 0.0
bag_topic_name: "grid_map"
